third-party/cmake/BuildLuarocks.cmake

```diff
@@ -42,7 +42,7 @@ endfunction()
 set(LUAROCKS_BINARY ${HOSTDEPS_BIN_DIR}/luarocks)
 
 # NOTE: Version must match version of LuaRocks in third-party/CMakeLists.txt
-set(LUAROCKS_VERSION 2.4)
+set(LUAROCKS_VERSION 2.4.3)
```

Comment:
Hmm maybe it doesn't need an update if only the patch number changed. Better leave this alone, unless CI complains.

Ultimately if CI passes then it's fine. I will also restart the CI after clearing the caches, before merging this.

Reply:
@concatime would you mind removing this change? Better to leave things unchanged if they aren't needed.